package.json

```diff
@@ -1,21 +1,24 @@
 {
-  "private": true,
-  "scripts": {
-    "build": "lerna run build",
-    "build:dev": "tsc --build",
-    "clean": "lerna run clean",
-    "install": "lerna bootstrap --hoist",
-    "publish": "npm run clean && npm run build && lerna publish --no-private -m \"Publish npm packages\"",
-    "test": "lerna run test",
-    "update:all": "update-dependency --lerna --minimal --regex .*",
-    "update:lab": "update-dependency --lerna --minimal --regex ^@jupyterlab/",
-    "update:lab:next": "update-dependency --lerna --minimal --regex ^@jupyterlab/ ^next",
-    "updated": "lerna updated",
-    "watch": "tsc --build --watch"
-  },
-  "devDependencies": {
-    "@jupyterlab/buildutils": "^3.0.0",
-    "lerna": "^4.0.0",
-    "rimraf": "^5.0.0"
-  }
+	"private": true,
```

Comment:
There are a lot of formatting changes in this PR. It would be very much easier for me to look at it if pure formatting changes were removed! 😃

Reply:
@vidartf That's right. Would you be ok to add a formatter like prettier on the project? This could be done in a separate PR. If so I can do it. The changes on migration would then be appear more clearly. Please let me know what you think about it ?

Reply:
@vidartf Thanks for your reply. So we can try to do what you are suggesting : " a config for prettier + an autoformat commit + the ignore config so that GH doesn't consider that commit for git blame." As a first step, I opened this draft PR https://github.com/jupyter/nbdime/pull/667 with the config for prettier (that can be tuned), and a commit with the autoformat changes. I guess that we can add in a second time a `.git-blame-ignore-revs` so that Github does not consider this commit for `git blame`. Let me please know if the prettier config does not fit your expectations.